packages/plugins/LoadingIndicator/README.md

```diff
@@ -28,9 +28,9 @@ map.$store.commit('plugin/loadingIndicator/removeLoadingKey', key)
 
 ![Loading indicator example](./readme_loadingIndicator_example.png)
 
-The key must be unique and is kept track of via a Set. It can't be added multiple times, and removing it once always removes it altogether. It is advised to use a key like `{my-plugin-or-application-name}-{procedure-name}` to avoid name conflicts. The LoadingIndicator will usually be used for asynchronous code.
+The key must be unique and is managed using a Set. This ensures that it can't be added multiple times, and removing it once will remove it completely. To avoid name conflicts, it is recommended to use a key format like `{my-plugin-or-application-name}-{procedure-name}`. The `LoadingIndicator` is typically used for asynchronous operations.
 
-As such, **always call `removeLoadingKey` in the `finally` section of your code** to prevent hanging loading indicators.
+Therefore, **always call `removeLoadingKey` in the `finally` block of your code** to prevent loading indicators from hanging.
```

Comment:
I see no added value in this paraphrasing and propose the change to this README is dropped completely together with the CHANGELOG change. I also perceive this section as to be alright as it was previously.

Should any substantial changes be made, the CHANGELOG should refer to both mutations, as the documentation change pertains to both mutations as indicated by the example above.

Reply:
@warm-coolguy 🏓